docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -0,0 +1,239 @@
+// Use these for links to issue and pulls.
+:kibana-issue: https://github.com/elastic/kibana/issues/
+:kibana-pull: https://github.com/elastic/kibana/pull/
+:beats-issue: https://github.com/elastic/beats/issues/
+:beats-pull: https://github.com/elastic/beats/pull/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.6.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.6.0 relnotes
+
+[[release-notes-8.6.0]]
+== {fleet} and {agent} 8.6.0
+
+Review important information about the {fleet} and {agent} 8.6.0 release.
+
+[discrete]
+[[breaking-changes-8.6.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+[discrete]
+[[breaking-1994]]
+.Each input in an agent policy must have a unique ID 
+[%collapsible]
+====
+*Details* +
+Each input in an agent policy must have a unique ID, like `id: my-unique-input-id`.
+This change only affects standalone agents. Unique IDs are automatically generated in
+agent policies managed by {fleet}. For more information, refer to
+{agent-pull}/1994[#1994]
+
+*Impact* +
+Make sure that your standalone agent policies have a unique ID.
+====
+
+[discrete]
+[[breaking-1140]]
+.Diagnostics `--pprof` argument has been removed and is now always provided
+[%collapsible]
+====
+*Details* +
+The `diagnostics` command gathers diagnostic information about the {agent} and
+each component/unit it runs. Starting in 8.6.0, the `--pprof`
+argument is no longer available because `pprof` information is now always
+provided. For more information, refer to {agent-pull}1140[#1140].
+
+*Impact* +
+Remove the `--pprof` argument from any scripts or commands you use.
+====
+
+[discrete]
+[[known-issues-8.6.0]]
+=== Known issues
+
+[discrete]
+[[known-issue-issue-2066]]
+.Osquery live query results can take up to five minutes to show up in {kib}
+[%collapsible]
+====
+*Details* +
+A known issue in {agent} may prevent live query results from being available
+in the {kib} UI even though the results have been successfully sent to {es}. 
+For more information, refer to {agent-issue}2066[#2066].
+
+*Impact* +
+Be aware that the live query results shown in {kib} may be stale.
```

Comment:
```suggestion
Be aware that the live query results shown in {kib} may be delayed by up to 5 minutes.
```

We can be more specific about the impact, we know that the results will be up to 5 minutes late because it is tied to the agent checkin period with the same duration.